run_tests.sh

```diff
@@ -18,17 +18,12 @@ REPO=dcrd
 
 go version
 
-# binary needed for RPC tests
-env CC=gcc go build
-cp "$REPO" "$(go env GOPATH)/bin/"
```

Comment:
Why is this being removed?  It is needed to ensure the latest dcrd is being used by the RPC tests in all environments.